packages/core/src/schema.ts

```diff
@@ -0,0 +1,3 @@
+export function createSchema<TSchema>(value?: any): TSchema {
+  return (value as any) as TSchema;
```

Comment:
`value` is already `any`, so you can just return it here

```suggestion
  return value;
```